CHANGELOG.md

```diff
@@ -1,31 +1,40 @@
 # Changelog
 
+All notable changes to this project will be documented in this file.
+
+The format is based on [Common Changelog](https://common-changelog.org/)
+and this project uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
 ## [Unreleased]
 
 ### Changed
 
 - Use built-in `assert` statements instead of `unittest` assertions in
   integration tests ([#743](https://github.com/nsidc/earthaccess/issues/743))
-  (@chuckwondo)
+  ([@chuckwondo](https://github.com/chuckwondo))
```

Comment:
Perhaps this is something to discuss, but I left out the explicit links because GitHub automatically understands the @ notation and implicitly links them for us. Dropping the explicit links would declutter the file for editing and make it quicker and easier to type someones handle without having to also type the link.

Reply:
@chuckwondo, that's a good point. I just matched the previous style, but I'm game to change it back. There are a couple of other nit-picks I have with the changelog formatting, so I'll open another PR for that.